### PR TITLE
pass a pointer in HandleFoodLoss to modify param "to"

### PR DESCRIPTION
**Pull request by Blackjack200**
# Tests
```patch
diff --git a/main.go b/main.go
index b1905144..35072aed 100644
--- a/main.go
+++ b/main.go
@@ -3,8 +3,6 @@ package main
 import (
 	"fmt"
 	"github.com/df-mc/dragonfly/server"
-	"github.com/df-mc/dragonfly/server/event"
-	"github.com/df-mc/dragonfly/server/player"
 	"github.com/df-mc/dragonfly/server/player/chat"
 	"github.com/pelletier/go-toml"
 	"github.com/sirupsen/logrus"
@@ -12,11 +10,6 @@ import (
 )
 
 func main() {
-	aa := a{}
-	to := 1
-	logrus.Infof("1 main: to:%v", to)
-	aa.HandleFoodLoss(event.C(), 20, &to)
-	logrus.Infof("2 main: to:%v", to)
 	log := logrus.New()
 	log.Formatter = &logrus.TextFormatter{ForceColors: true}
 	log.Level = logrus.DebugLevel
@@ -36,16 +29,6 @@ func main() {
 	}
 }
 
-type a struct {
-	player.NopHandler
-}
-
-func (a) HandleFoodLoss(_ *event.Context, from int, to *int) {
-	logrus.Infof("1 HandleFoodLoss: from:%v to:%v", from, *to)
-	*to = 0
-	logrus.Infof("2 HandleFoodLoss: from:%v to:%v", from, *to)
-}
-
 // readConfig reads the configuration from the config.toml file, or creates the
 // file if it does not yet exist.
 func readConfig(log server.Logger) (server.Config, error) {

```

## Excepted Result
```
INFO[0000] 1 main: to:1
INFO[0000] 1 HandleFoodLoss: from:20 to:1
INFO[0000] 2 HandleFoodLoss: from:20 to:0
INFO[0000] 2 main: to:0
```

## Actual Result
(no difference)
```
INFO[0000] 1 main: to:1
INFO[0000] 1 HandleFoodLoss: from:20 to:1
INFO[0000] 2 HandleFoodLoss: from:20 to:0
INFO[0000] 2 main: to:0
```

### CODE REVIEW

File: server/player/handler.go [hunk 1]
```diff
@@ -163,10 +163,9 @@ func (NopHandler) HandleItemConsume(*event.Context, item.Stack)
 func (NopHandler) HandleItemDamage(*event.Context, item.Stack, int)                           {}
 func (NopHandler) HandleAttackEntity(*event.Context, world.Entity, *float64, *float64, *bool) {}
 func (NopHandler) HandleExperienceGain(*event.Context, *int)                                  {}
-func (NopHandler) HandlePunchAir(*event.Context)                                              {}
```
Comment (Flonja) [inline server/player/handler.go:166]: This seems like a change that didn't need to get made

File: server/player/handler.go [hunk 2]
```diff
@@ -163,10 +163,9 @@ func (NopHandler) HandleItemConsume(*event.Context, item.Stack)
...
 func (NopHandler) HandleHurt(*event.Context, *float64, *time.Duration, world.DamageSource)    {}
 func (NopHandler) HandleHeal(*event.Context, *float64, world.HealingSource)                   {}
-func (NopHandler) HandleFoodLoss(*event.Context, int, int)                                    {}
-func (NopHandler) HandleDeath(world.DamageSource, *bool)                                      {}
+func (NopHandler) HandleFoodLoss(*event.Context, int, *int)                                   {}
+func (NopHandler) HandleDeath(world.DamageSource, *bool)                                                 {}
```
Comment (Flonja) [inline server/player/handler.go:169]: Just like this extra whitespace

**Review by Flonja — CHANGES_REQUESTED** [review]
Just some small changes

**Review by Sandertv — COMMENTED** [review]
The build doesn't currently pass because of the reasons stated above by @Flonja. This should be ready to go once the build issues are fixed.

**Pull request merged**